event/aggregator.go

```diff
@@ -35,11 +37,12 @@ type Result struct {
 type EventAggregator struct {
 	examples bool
 	// --
-	result *Result
+	result    *Result
+	tzDiffUTC time.Duration
```

Comment:
I suggest renaming this to something more standard: `utcOffset`. Afaik that's an established term.